e2e/various-suite/loki-table-explore-to-dash.spec.ts

```diff
@@ -110,7 +110,6 @@ const lokiQueryResult = {
   },
 };
 
-// Skipping due to race conditions with same old arch test e2e/various-suite/loki-table-explore-to-dash.spec.ts
 describe.skip('Loki Query Editor', () => {
```

Comment:
can we unskip this test now then? 🤔

Reply:
removed comment but forgot to remove `.skip` 🤦‍♂️